tests/src/Test/Tasty/Clash.hs

```diff
@@ -241,6 +241,10 @@ instance IsTest ClashGenTest where
       , "-odir", oDir
       , "-hidir", oDir
       , "-fclash-debug", "DebugSilent"
+
+      -- PR #2511 introduced a lot of CPP in shouldwork/*. To prevent very tedious
+      -- work, we enable CPP by default.
```

Comment:
How ironic that this comment about tedious CPP work now generates a CPP error

Reply:
That's hilarious haha